lib/persona-ruby/actions/inquiries.rb

```diff
@@ -22,6 +22,10 @@ def inquiry_update(id, attributes = {})
       def inquiry_delete(id)
         connection.delete("inquiries/#{id}", {})
       end
+
+      def inquiry_generate_one_time_link(id, expiration_time = 3600)
```

Comment:
Maybe do `expiration_time: 3600` instead so we can add hash like options in the future if needed.